src/tasks/helpers/getContainer.js

```diff
@@ -96,23 +98,25 @@ var augmentModule = function(modulesOutput, extensionName, extensionPath, module
   });
 };
 
-var augmentModules = function(extensionOutput, extensionDescriptor, extensionPath) {
+const augmentModules = function(extensionOutput, extensionDescriptor, extensionPath) {
   extensionOutput.modules = extensionOutput.modules || {};
 
   FEATURE_TYPES.forEach(function(featureType) {
     if (extensionDescriptor.hasOwnProperty(featureType)) {
-      var featureDescriptors = extensionDescriptor[featureType];
+      const featureDescriptors = flatten([extensionDescriptor[featureType]]);
```

Comment:
I don't really understand the purpose of`flatten` here. Would this accomplish the same goal or is there a nested array somewhere I'm unaware of?

```
let featureDescriptors = extensionDescriptor[featureType];

if (!Array.isArray(featureDescriptors)) {
  featureDescriptors = [featureDescriptors]
}
```